dbt_subprojects/hourly_spellbook/models/_project/cow_protocol/arbitrum/cow_protocol_arbitrum_eth_flow_orders.sql

```diff
@@ -0,0 +1,62 @@
+{{  config(
+        alias='eth_flow_orders',
+
```

Comment:
plz include schema property here too. it makes it much easier to follow table config in future 🙏 
```suggestion
schema='cow_protocol_arbitrum',
```